figenv.py

```diff
@@ -106,6 +116,10 @@ def __getattr__(cls, name):
         if override_via_environment and prefix + name in os.environ:
             value = os.environ[prefix + name]
 
+        if value == _MISSING:
```

Comment:
This should be an `is` instead of `==` since we are comparing to an exact instance of an object.